lib/DocumentFilter/DocumentFilterRule.test.js

```diff
@@ -51,52 +54,65 @@ describe('DocumentFilterRule', () => {
     await Button('Submit').exists();
   });
 
-  it('renders Name option', () => {
+  test('renders the correct options for the Attribute dropdown', async () => {
     TextArea('Name').exists();
-  });
-
-  it('renders Note option', () => {
     TextArea('Note').exists();
-  });
-
-  it('renders Physical location option', () => {
+    TextArea('Category').exists();
     TextArea('Physical location').exists();
-  });
-
-  // TODO this is not how we check select options
-  it('renders URL option', () => {
     TextArea('URL').exists();
-  });
-
-  it('renders Content type option', () => {
     TextArea('Content type').exists();
-  });
-
-  it('renders File name option', () => {
     TextArea('File name').exists();
   });
 
-  it('renders Category option', () => {
-    TextArea('Category').exists();
-  });
-
-  it('renders is option', () => {
+  test('renders the correct options for the Comparator dropdown', async () => {
```

Comment:
Same thing as above though... pleasee keep individual checks in their own test/it